efel/tests/test_cppcore.py

```diff
@@ -154,3 +154,22 @@ def test_getFeature_non_existant(self):
         """cppcore: Testing failure exit code in getFeature"""
         import efel.cppcore
         efel.cppcore.getFeature("does_not_exist", list())
+
+    def test_registerFeature(self):
+        import efel.cppcore
+
+        def getTwiceAP_Amp(output):
```

Comment:
That's mainly used as an internal cache, correct?  
I could keep that local to cppcore.cpp, if that makes sense.  What is the invalidation policy for the cache at the moment (so I can mirror it)?